src/index.html

```diff
@@ -1,27 +1,446 @@
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="page"
+>
   <head>
     <meta charset="UTF-8" />
+
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+
     <title>Catalog</title>
+
     <link
       rel="preconnect"
       href="https://fonts.gstatic.com"
     />
+
     <link
       href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
       rel="stylesheet"
     />
+
     <link
       rel="stylesheet"
       href="styles/index.scss"
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="logo">
+        <a
+          href="https://"
```

Comment:
ur link is broken.
fix it
```suggestion
          href="#"
```